selftests/runners/test_lisa_runner.py

```diff
@@ -632,7 +633,10 @@ def _run_all_tests(self, runner: LisaRunner) -> List[TestResult]:
         while not runner.is_done:
             task = runner.fetch_task()
             if task:
-                temp_test_results = task()
+                if isinstance(task, Task):
+                    temp_test_results = task()
+                if isinstance(task, List):
```

Comment:
`elif`

Reply:
This is also present in the runner file. I will fix it there as well